docs/getting-started/setup/using-bundlers.md

```diff
@@ -0,0 +1,116 @@
+---
+category: setup
+meta-title: Using bundlers | CKEditor 5 documentation
+order: 130
+modified_at: 2024-08-30
+---
+
+# Using bundlers to bundle CKEditor&nbsp;5
```

Comment:
The heading is shortened in the table of contents. Maybe we could add a menu title or make the heading shorter.
```suggestion
# Using bundlers to bundle the editor.
```